### PR TITLE
Expanded controller

### PR DESCRIPTION
**Pull request by jcarolus**
* option to show the expanded media controls

The mini controller was causing some headache, so for now this seems a good improvement.
It's not clear to me why it won't show images from metadata - the documentation on this is vague.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +49,7 @@ const appComponents = [
   'tns-core-modules/ui/frame/activity',
   'nativescript-cast/cast-activity',
   'nativescript-cast/cast-options-provider',
+  'nativescript-cast/extended-controller-activity'
```
Comment (lochstar) [inline README.md]: Should this be `expanded-controller-activity` ?

File: demo/app/main-view-model.ts [hunk 2]
```diff
@@ -1,10 +1,11 @@
-import { Observable } from 'tns-core-modules/data/observable';
-import { EventData } from 'tns-core-modules/ui/core/view';
-import { CastEvent, CastMediaInfo, CastMediaStatus } from 'nativescript-cast/cast.types';
+import {Observable} from 'tns-core-modules/data/observable';
```
Comment (lochstar) [inline demo/app/main-view-model.ts]: Can we keep the spaces around the brackets `{ Observable }`. It's probably your linter formatting it automatically.

**Comment by jcarolus** [comment]
Just discovered it crashes on Android after a while.
Also not happy with imlementation. Will be continued...

**Comment by jcarolus** [comment]
The crash should be fixed by the change to AndroidManifest.
It's indeed code formatting default of my editor, so added those spaces back.

**Comment by jcarolus** [comment]
@lochstar I have added demo/webpack.config.js to the repository, it needed to be upgraded for latest nativescript and it's convenient to have the changes for Android in there anyway. 
A bit in doubt if it should be in the repo (the plugin template exludes it as well), but it does need some editing. What is your opinion?

**Comment by lochstar** [comment]
> @lochstar I have added demo/webpack.config.js to the repository,

Ouch it's meant to be there but has been ignored by the the `.gitignore` rule `*.js`. I'll get it fixed up.

**Comment by jcarolus** [comment]
@lochstar another small issue; the image did not show up because  the image in the metadata `https://d1u5p3l4wpay3k.cloudfront.net/lolesports_gamepedia_en/2/24/Space_eSportslogo_square.png?version=1352e7508b7e001da75af441b9221997` is no longer reachable.
Perhaps also a via github.io hosted image?

So in my opinion ready to merge :) !

**Comment by lochstar** [comment]
Good stuff!

**Pull request merged**